.github/workflows/deps_lint.yml

```
@@ -200,13 +200,26 @@ jobs:
       if: matrix.python-version == 3.8
       run: pytest -rs --cov=./optimade/ --cov-report=xml --cov-append tests/adapters/
 
+    - name: Run tests for validator only to assess coverage
+      if: matrix.python-version == 3.8
+      run: pytest -rs --cov=./optimade/ --cov-report=xml:validator_cov.xml tests/server/test_server_validation.py
+
     - name: Upload coverage to Codecov
       if: matrix.python-version == 3.8 && github.repository == 'Materials-Consortia/optimade-python-tools'
       uses: codecov/codecov-action@v1
       with:
+        name: project
         file: ./coverage.xml
         flags: unittests
 
+    - name: Upload coverage to Codecov
+      if: matrix.python-version == 3.8 && github.repository == 'Materials-Consortia/optimade-python-tools'
+      uses: codecov/codecov-action@v1
+      with:
+        name: validator
+        file: ./validator_cov.xml
+        flags: unittests
```

I think we should change the flags both here and the previous step.
I should've done this when converting to pytest.
```suggestion
        flags: pytest
```

---

Yep, fair enough. I've enabled a separate codecov report/upload for the code that is hit by the validator, should give us a reasonable proxy for the bits that we're forgetting/unable to validate (from our implementation, at least).

---

Yeah, I see that. And that's indeed a great idea!